fala/apps/adviser/tests/test_crown_dependencies_playwright.py

```diff
@@ -0,0 +1,24 @@
+from playwright.sync_api import expect
+from fala.playwright.setup import PlaywrightTestSetup
+
+
+class CrownDependenciesTest(PlaywrightTestSetup):
+    def test_jersey(self):
+        results_page = self.visit_results_page("JE1")
+        expect(results_page.h1).to_have_text("The postcode JE1 is in Jersey")
+        search_page = results_page.change_search()
```

Comment:
this is confusing. Based on the `def visit_results_page` I don't think they should be able to change search here, right? Or maybe it is a slightly different change button? I wonder if that could/should be clearer.

Reply:
`change_search()` is looking for the "Change search" text. 

- There is a "Change search" button on ResultsPage with #changeSearchButton
- There is a "Change search" button on OtherRegionPage with #otherRegionChangeSearchButton

I think test is clear, but will add the above as a comment on the method.